backend/wmg/data/utils.py

```diff
@@ -207,3 +208,21 @@ def to_dict(a, b):
     slists = [b[bounds_left[i] : bounds_right[i]] for i in range(bounds_left.size)]
     d = dict(zip(np.unique(a), [list(set(x)) for x in slists]))
     return d
+
+
+def get_pinned_ontology_url(name: str):
+    """
+    This function retrieves the URL of the pinned ontology based on the provided name.
+
+    Parameters:
+    name (str): The name of the ontology (e.g. cl-basic.obo).
+
+    Returns:
+    str: The URL of the pinned ontology.
+    """
+
+    response = requests.get(CL_PINNED_CONFIG_URL)
+    decoded_yaml = yaml.safe_load(response.content.decode())
+    cl_url = list(decoded_yaml["CL"]["urls"].values())[0]
```

Comment:
there's no guarantee the latest value will always be the first in the list. I'd grab `key = decoded_yaml["CL"]["latest"]` and then `cl_url = decoded_yaml["CL"]["URLs"][key]`, then do the splitting